Week1/homework/app.js

```diff
@@ -1,11 +1,182 @@
+    },
+    the_odyssey: {
+      author: 'Homer',
+      name: 'The Odyssey',
+      alt: 'This is the cover of the book of Verlag Der called The Odyssey',
+      description:
+        'The Odyssey narrates the path of a fascinatingly complex hero through a world of wonders and danger-filled adventure.'
+    },
+    death_in_venice: {
+      author: 'Thomas Mann',
+      name: 'Death in Venice',
+      alt: 'This is the cover of the book called Death in Venice',
+      description:
+        'One of the most famous literary works of the 20th century, the novella "Death in Venice" embodies themes that preoccupied Thomas Mann (1875–1955) in much of his work.'
+    },
+    adventures_of_huckleberry_finn: {
+      author: 'Mark Twain',
+      name: 'Adventures of Huckleberry Finn',
+      alt: 'This is the cover of the book called Adventures of Huckleberry Finn',
+      description:
+        "The novel's preeminence derives from its wonderfully imaginative re-creation of boyhood adventures along the Mississippi River."
+    },
+    oliver_twist: {
+      author: 'Charles Dickens',
+      name: 'Oliver Twist',
+      alt: 'This is the cover of the book called Oliver Twist',
+      description:
+        'A profound social critic, Dickens introduced genteel readers to the problems of the poor in a way that had rarely been attempted before.'
+    },
+    frankenstein: {
+      author: 'Mary Shelley',
+      name: 'Frankenstein',
+      alt: 'This is the cover of the book called Frankenstein',
+      description:
+        "The story of Victor Frankenstein's terrible creation and the havoc it caused has enthralled generations of readers and inspired countless writers of horror and suspense. "
+    },
+    return_of_native: {
+      author: 'Thomas Hardy',
+      name: 'The Return of the Native',
+      alt: 'This is the cover of the book called The Return of the Native',
+      description:
+        'It first appeared in the magazine Belgravia, a publication known for its sensationalism, and was presented in twelve monthly installments from January to December 1878.'
+    }
+  };
+
+  const image = {
+    home_and_world: './images/home_and_world.jpeg',
+    the_rescue: './images/the_rescue.jpeg',
+    heart_of_darkness: './images/heart_of_darkness.jpeg',
+    country_of_pointed_fires: './images/country_of_pointed_fires.jpeg',
+    howards_end: './images/howards_end.jpeg',
+    robinson_crusoe: './images/robinson_crusoe.jpeg',
+    the_odyssey: './images/the_odyssey.jpeg',
+    death_in_venice: './images/death_in_venice.jpeg',
+    adventures_of_huckleberry_finn: './images/adventures_of_huckleberry_finn.jpeg',
+    oliver_twist: './images/oliver_twist.jpeg',
+    frankenstein: './images/frankenstein.jpeg',
+    return_of_native: './images/return_of_native.jpeg'
+  };
+
+  const div = document.getElementById('content');
```

Comment:
Hi @YaseirAlkhwalda - I think you missed something listed in the requirements. This may not be functionally important but I believe such things would help you becoming a better developer. For example, **1.5** and **1.8**  state that you modify/create functions to achieve the desired functionality.

The current piece of code written works just fine but it is a good practice to separate out your code into smaller functions. Functions provide a lot of benefits like re-usable code and more semantic understanding of the code. 

For example, you can have a function(say `createAppLayout`) to create the layout of your application with the `content`, `h1`, `div` and `ul` tags. Similarly, you can create a function(say `createBookItems`) for **1.5** where you add all the books from your object as li elements to the `ul` tag. And at the end, have another function(say `addBookImages`) for **1.8** where you use your `images` object to add images to all books that you created in the previous function.

I would suggest that you give those two requirements another go to split out your code into smaller functions. 🙂